tests/integration/test_integration.py

```diff
@@ -769,42 +769,92 @@ def test_search_revision(self, admin_client, list_name, list_c):
         assert len(results) == 1
         assert results[0].record_list.identifier == list_c.identifier
 
-    def test_search_by_database(self, admin_client, list_name, unresolvable_item, list_c):
+    @pytest.mark.parametrize("include_items", [True, False])
```

Comment:
These three searches now test with `include_items` True and False, and check that the results are as expected, only down to the level of having a list of items of the appropriate length. The final test in this module now only checks that the items are correct, instead of testing the broader 'include_items' behavior.

There's no significant change to the test performance, because the fixtures are all reused